### PR TITLE
Explain that cmctl x uninstall is now safe by default

### PR DESCRIPTION
**Pull request by inteon**
This PR adds the documentation for the feature that was added in https://github.com/cert-manager/cmctl/pull/13.

### CODE REVIEW

File: content/docs/reference/cmctl.md [hunk 1]
```diff
@@ -329,8 +332,6 @@ cmctl x uninstall
 cmctl x uninstall --namespace my-cert-manager
 
 cmctl x uninstall --dry-run
-
-cmctl x uninstall --no-hooks
```
Comment (inteon) [inline content/docs/reference/cmctl.md:333]: unrelated change: this flag is no longer supported since v2.0.0

File: content/docs/reference/cmctl.md [hunk 2]
```diff
@@ -317,7 +317,10 @@ cmctl x uninstall
 
 This command uninstalls any Helm-managed release of cert-manager.
 
-The CRDs will be deleted if you installed cert-manager with the option `--set CRDs=true`.
+Starting from cmctl `v2.0.0`, the uninstall command is safe and will not delete the CRDs
+by default, even if they were installed with the Helm chart (using the option `--set installCRDs=true`).
+
+> 👴 Before `v2.0.0`, cmctl would remove the CRDs if they were installed with the Helm chart (similar to Helm's behavior).
```
Comment (wallrj) [inline content/docs/reference/cmctl.md:323]: Not sure what the old man icon is about :-) Typo?

**Review by wallrj — CHANGES_REQUESTED** [review]

File: content/docs/reference/cmctl.md (hunk 2 above)
Comment (inteon) [reply content/docs/reference/cmctl.md:323]: I thought that would be icon that indicates that this doc is for an old version.

Comment (inteon) [reply content/docs/reference/cmctl.md:323]: Maybe 🕐 could be used instead?

Comment (wallrj) [reply content/docs/reference/cmctl.md:323]: Oh. Ok. If it was intentional, then I'm happy to go with that.

/lgtm
/hold because I think the 🕐 does look better.

Comment (inteon) [reply content/docs/reference/cmctl.md]: I updated the icon.

**Review by wallrj — APPROVED** [review]
Thanks.

/lgtm
